### ISSUE TITLE
laion/larger_clap_music returns near-constant text embeddings

### ISSUE THREAD

**Issue by hubielu** [embeddings-benchmark/mteb#5069]
### Describe the bug

The model laion/larger_clap_music is registered in mteb/models/model_implementations/clap_models.py, but its text tower returns essentially the same embedding regardless of input. Any retrieval or zero-shot result produced with this model is therefore meaningless. Comparing all three LAION CLAP checkpoints on identical code, the same transformers version, the same processor, and the same four unrelated input strings: for laion/larger_clap_music the CLS hidden-state standard deviation is 0.025, the pooler output standard deviation is 0.005, and the mean pairwise text-text cosine similarity is 0.9993. For laion/clap-htsat-unfused those same three numbers are 0.526, 0.329 and 0.180. For laion/larger_clap_general they are 0.567, 0.441 and 0.294. In other words, the two working checkpoints show roughly 20x more variance in their text representations, and produce sensibly distinct embeddings for distinct inputs. larger_clap_music collapses all inputs to a single direction. Expected behaviour is that unrelated captions embed to distinct vectors, as they do for the other two checkpoints.

### To reproduce

Run the following. Every entry of the printed similarity matrix comes out at approximately 1.0.

import torch
from transformers import ClapModel, ClapProcessor
from transformers.modeling_outputs import BaseModelOutputWithPooling

m = ClapModel.from_pretrained("laion/larger_clap_music").eval()
p = ClapProcessor.from_pretrained("laion/larger_clap_music")
ts = ["a heavy metal guitar solo", "a slow sad piano ballad",
      "a female opera singer", "fast electronic dance music"]
with torch.no_grad():
    o = m.get_text_features(**p(text=ts, return_tensors="pt", padding=True, truncation=True))
    e = o.pooler_output if isinstance(o, BaseModelOutputWithPooling) else o
e = torch.nn.functional.normalize(e, dim=-1)
print(e @ e.T)

Substituting laion/clap-htsat-unfused or laion/larger_clap_general in both from_pretrained calls gives normal, varied similarities.

### Additional information

Reproduced on both transformers 4.57.6 and 5.14.1, so this is not a version regression. On an audio-text retrieval task the model scores below the random baseline, at 0.0133 and 0.0167 hit_rate@5 for the audio-to-text and text-to-audio directions respectively, against a random-encoder baseline of 0.0233 and 0.0192. The other two CLAP checkpoints score between 0.21 and 0.29 on the same task. Found while adding a music retrieval dataset for issue #2683.

### Are you interested to contribute a fix for this bug?

Yes

**Comment by Samoed** [comment]
I don't think we can do anything with this and this is model specific problem
